.github/workflows/test.yaml

```diff
@@ -23,24 +24,31 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.7", "3.8", "3.9", "3.10"]
-        dep-installer: ["dev", "pypi"]
-    continue-on-error: ${{ matrix.dep-installer == 'pypi' }}
 
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout VLSIR Repo
+        uses: actions/checkout@v3
 
       - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v2
-        with:
-          python-version: ${{ matrix.python-version }}
+        run: |
+          apt-get install -y python${{ matrix.python-version }}
+          apt-get install -y python${{ matrix.python-version }}-dev
+          apt-get install -y python${{ matrix.python-version }}-venv
+          apt-get install -y python${{ matrix.python-version }}-distutils
 
-      - name: Install dependencies
-        run: bash scripts/install-${{ matrix.dep-installer }}.sh
+      #! The bizarre insanity of ". ./venv/..." is an Ubuntu sh translation of
+      #! "source venv/..." seen in most Python documentation
 
-      - name: Test with pytest
-        run: |
-          pytest -sv --cov=./ --cov-report=xml
+      - name: update `pip`
+        run: python${{ matrix.python-version }} -m pip install --upgrade pip
 
+      - name: pytest `HDL21`
```

Comment:
We also want to keep the "test with either PyPi or GitHub versions of VLSIR" option that was removed from here.